src/components/timeline/TrackMemory.js

```diff
@@ -27,7 +27,7 @@ import type {
 
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
 
-import './TrackMemory.css';
+import 'firefox-profiler/components/timeline/TrackMemory.css';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.